cmd/kots/cli/backup.go

```diff
@@ -23,20 +26,37 @@ func BackupCmd() *cobra.Command {
 
 			namespace := v.GetString("namespace")
 
+			output := v.GetString("output")
+			if output != "json" && output != "" {
+				return errors.Errorf("output format %s not supported (allowed formats are: json)", output)
+			}
+
 			options := snapshot.CreateInstanceBackupOptions{
 				Namespace: namespace,
 				Wait:      v.GetBool("wait"),
+				Silent:    output != "",
 			}
-			if err := snapshot.CreateInstanceBackup(cmd.Context(), options); err != nil {
+			backupRes, err := snapshot.CreateInstanceBackup(cmd.Context(), options)
+			if err != nil && output == "" {
```

Comment:
what about if there's an error and an output format is specified? i don't see that handled

Reply:
should be like this one :) https://github.com/replicatedhq/kots/pull/2083/files#diff-5e5da03ddf8e728ae099106e99cc9a821f479eeaaedee622042cf98baae91626R56

Reply:
ahhh i had already fixed that but must've lost it while splitting. good catch!

Reply:
fixed